src/components/Table.jsx

```diff
@@ -1,22 +1,21 @@
 import * as React from "react";
 import { DataGrid } from "@mui/x-data-grid";
-import style from "./Table.module.css";
-class DataGridDemo extends React.Component {
-  render() {
-    return (
-      <div style={{ height: 300, width: "90%" }}>
-        <DataGrid
-          className={style.tableBackground}
-          rows={this.props.valueRows}
-          columns={this.props.valueColumns}
-          pageSize={4}
-          rowsPerPageOptions={[5]}
-          checkboxSelection
-          disableSelectionOnClick
-        />
-      </div>
-    );
-  }
+import style from "../styles/Table.module.css";
+
+function DataGridDemo(props) {
+  return (
+    <div id={style.tableSize}>
+      <DataGrid
+        className={style.tableBackground}
+        rows={props.rows}
```

Comment:
It would be great to use destructuring like in the link for cleaner code : https://javascript.plainenglish.io/destructure-react-props-with-es6-object-destructuring-for-cleaner-code-3984453e484d
So you will have something like this below and simply just use rows and columns later everywhere :) 
const { rows, columns } = props;